src/Cli/func/Helpers/GlobalCoreToolsSettings.cs

```diff
@@ -99,6 +99,13 @@ public static void Init(ISecretsManager secretsManager, string[] args)
             {
                 _currentWorkerRuntime = WorkerRuntime.None;
             }
+
+            if (_currentWorkerRuntime == WorkerRuntime.Java || args.Any(arg => string.Equals(arg, WorkerRuntime.Java.ToString(), StringComparison.OrdinalIgnoreCase)))
+            {
+                ColoredConsole
+                    .WriteLine(WarningColor($"This action is not supported when using the core tools directly. Please use one of the supported environments to test {_currentWorkerRuntime} apps locally as specified by" +
```

Comment:
"Core Tools" capitalized.

Also, is this really about "supported environments" or even "testing"? It's more about scaffolding templates, right? Or I guess it could be multiple scenarios? It might be easier to just treat it as "work with".

```suggestion
                    .WriteLine(WarningColor($"This action is not supported when using the Core Tools directly. Please see {link} for instructions on working with {_currentWorkerRuntime} apps locally." +
```

Reply:
I'm also questioning if this should be warning level in all cases. It feels like this should be an error for some actions, right?

Reply:
It's definitely an error for some actions -- we could also conditionally log warnings/errors based on the type of action. This is just the file location we landed on during design since all of the impacted commands would come through here.